### ISSUE TITLE
SmartGit Plugin

### ISSUE THREAD

**Issue by arnaud-m** [folkswithhats/fedy#449]
Dear Fedy devs,

I have noticed that the SmartGit plugin was broken in Fedy 4.2.5. 

I have fixed it the best I can. 

1. I fixed the FILE extraction: (head+grep+tr) -> sed
2. I fixed the URL format

```sh
FILE=smartgit-linux-$(wget "https://www.syntevo.com/smartgit/changelog.txt" -O - |  sed -n '1{s/SmartGit \([0-9][0-9]*\)\.\([0-9][0-9]*\)\.\([0-9][0-9]*\) .*/\1_\2_\3/ ; p}').tar.gz
URL=http://www.syntevo.com/static/smart/download/smartgit/$FILE
```

It works at least for me.
Cheers

**Comment by bddenhartog** [comment]
Thank you for the report, we'll get this patched momentarily.

**Comment by bddenhartog** [comment]
Sorry for the delay - things came up and I totally forgot about this.

**Comment by Abhinav1217** [comment]
Solved in latest PR.

**Comment by bddenhartog** [comment]
Closed in 0095bab5.
